discovery-infra/test_infra/controllers/node_controllers/host.py

```diff
@@ -63,6 +63,14 @@ def kill_service(self, service):
         logging.info("Killing service %s on host %s", service, self.name)
         self.run_command(f'sudo systemctl kill {service}.service || true')
 
+    def kill_installer(self):
+        logging.info("Killing installer on host %s", self.name)
+        output = self.run_command("sudo su root -c '"
```

Comment:
I think , that for debuggablitiy porposes, first grep and print that container, and then kill it. because later on, if something we'll go wrong, you will not know if u actually killed it or it was never was running in the first place

Reply:
lets have it more generic, kill_container and just provide name